docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -2,16 +2,20 @@
 
 <p class="description">This guide describes the changes needed to migrate the Data Grid from v5 to v6.</p>
 
-## Start using the alpha release
+## Start using the new release
 
-In `package.json`, change the version of the data grid package to `next`.
+In `package.json`, change the version of the data grid package to `latest` or `^6.0.0`.
 
 ```diff
--"@mui/x-data-grid": "latest",
-+"@mui/x-data-grid": "next",
+-"@mui/x-data-grid": "5.X.X",
++"@mui/x-data-grid": "latest",
 ```
 
-Using `next` ensures that it will always use the latest v6 alpha release, but you can also use a fixed version, like `6.0.0-alpha.0`.
+Using `latest` ensures that it will always use the latest v6 release, but you can also use a fixed version, like `6.0.0`.
```

Comment:
> Using `latest` ensures that it will always use the latest v6 release

Not 100% correct - it will use v7 once it's released.
We can remove this statement completely as it doesn't seem to bring any additional value.